src/components/AdministrationPage/AdministrationPage.tsx

```diff
@@ -23,6 +30,28 @@ export const AdministrationPage = () => {
     paddingRight: '8px',
     paddingBottom: '5px',
   };
+  const [secondsUntilReload, setSecondsUntilReload] = useState<number>(REFRESH_INTERVAL_SECONDS);
+  const dataContainer = useDataContainer(
+    useCallback(({ requestConfig }: IService) => buildService.getBuildCount(requestConfig), [])
+  );
+  const dataContainerRefresh = dataContainer.refresh;
+  // TODO: Create a better solution than disabling the next line
+  useEffect(() => dataContainer.refresh({}), []); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
`runImmediately` (useInterval) can be used instead of this line?

Reply:
Does not work as expected, triggers an instant countdown of one second. Do we want to proceed anyway?